### DISCUSSION TITLE
Layout Jumping

### DISCUSSION THREAD

**Discussion by TheDarkStrix** [vercel/next.js#23500] [Help] [answered]
I made a website using NEXTJS , I came across this issue . I am not sure if its related to nextjs or not.

The Issue:
Linked below is a page , with a very basic layout . On page change using the nav links above, notice how the layout jumps.
This is weird not sure why this is happening.

Other Tools used here,
Framer-motion - To make the page transition.
Reactstrap 

Any help regarding is very much appreciated.  

Deployed Link : https://demo-vert-delta.vercel.app/
Github Repo : https://github.com/TheDarkStrix/demo

**Comment by wwwdepot** [comment, marked as answer]
Use `<AnimatePresence exitBeforeEnter>` in [`_app.js`](https://github.com/TheDarkStrix/demo/blob/f8f22e052eee238fd4109ec26f3d7ee43ef4373e/pages/_app.js#L6)

**Reply by TheDarkStrix** [reply]
thanks !
